src/scales/scale.time.js

```diff
@@ -676,10 +677,11 @@ module.exports = Scale.extend({
 		var minorFormat = formats[me._unit];
 		var majorUnit = me._majorUnit;
 		var majorFormat = formats[majorUnit];
-		var majorTime = +adapter.startOf(time, majorUnit);
+		var tick = !isNaN(index) && ticks[index];
```

Comment:
Why do we need to test `index`? I would expect a valid number (else it's a bug).